### PR TITLE
userinfo not showing fixed and signout route fixed

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/services/api.js [hunk 1]
```diff
@@ -124,7 +135,7 @@ class MayaApiService extends BaseApiService {
             'Content-Type': 'application/json',
             'Accept': 'application/json',
             'Flavour': flavour,
-            'PSAuthKey': authToken,
+            // 'PSAuthKey': authToken,
```
Comment (arrogant-ant) [inline src/services/api.js:138]: remove this code

File: vue.config.js [hunk 2]
```diff
@@ -2,10 +2,11 @@ const zlib = require('zlib');
 const PrerenderSPAPlugin = require('prerender-spa-plugin');
 const path = require('path');
 module.exports = {
-    devServer: {
-        host: 'parkspot.localhost',
-        proxy: 'https://www.parkspot.in',
-    },
+    // was working but now not working why?
```
Comment (arrogant-ant) [inline vue.config.js:5]: do we need this change?

**Review by arrogant-ant — APPROVED** [review]

File: vue.config.js (hunk 2 above)
Comment (alias01) [reply vue.config.js:5]: it was needed when we are not able to test firebase in local due to authorised domain... now I have whitelisted localhost ..so not needed.. also it stopped working as well.

File: src/services/api.js (hunk 1 above)
Comment (alias01) [reply src/services/api.js:138]: done

**Pull request merged**